.github/workflows/reset-preview-db.yml

```
@@ -0,0 +1,61 @@
+          if [ "$HTTP_STATUS" -ge 200 ] && [ "$HTTP_STATUS" -lt 300 ]; then
+            echo "Preview branch reset successfully."
+          else
+            echo "::warning::Preview branch reset failed (HTTP ${HTTP_STATUS})."
+            echo "Response:"
+            cat /tmp/neon-response.json
+            exit 1
+          fi
+
+      - name: Post summary
+        if: always()
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:1:31: Double quote to prevent globbing and word splitting

(shellcheck)